jshift-kit-quarkus/src/main/java/io/jshift/quarkus/generator/QuarkusGenerator.java

```diff
@@ -38,7 +38,7 @@
 public class QuarkusGenerator extends BaseGenerator {
 
     public QuarkusGenerator(GeneratorContext context) {
-        super(context, "quarkus");
+        super(context, "io/jshift/quarkus/generator");
```

Comment:
Need to short this namespace.

Reply:
aah. intellij does this auto refactor. I missed to check this again

Reply:
fixed